### DISCUSSION TITLE
Features suggestion

### DISCUSSION THREAD

**Discussion by ghost** [UmerCodez/SensorServer#5] [Ideas]
It would be great if the server wasn't shutting down after we close the app.
And a "start on boot" option would be useful too.

(sorry if opening an issue wasn't necessary for this, I don't really know how github works.)

**Comment by UmerCodez** [comment]
@n0n467 
Thanks for sharing new idea. 

Currently both of your suggestions can be implemented by non trivial changes but the question is why would some one want the server to start on boot up.  What's the use case ?

Secondly, if you switch to other app while SensorSever is running (not closed) , the connected clients still receives sensor data.

(Github offers [discussions](https://github.com/umer0586/SensorServer/discussions) for non issues stuff. Issue can easily be converted to discussions )

**Comment by johanjohan** [comment]
@umer0586 great app. Would it be possible to include GPS position data lat lon alt?

**Reply by UmerCodez** [reply]
@johanjohan Yes its possible. I am planning to add this feature when i have time.

**Comment by Lu-Yahui** [comment]
Is streaming camera data in plan?

**Reply by UmerCodez** [reply]
> Is streaming camera data in plan?

This app primarily focuses on streaming sensor data from various sensors on an Android device. While camera data streaming is not a central feature of this app, it may not be included as a core functionality.

**Comment by dcases** [comment]
Can you include a battery state data? Its level and if it's charging or not

**Reply by UmerCodez** [reply]
Sure, showing battery level and charging status is a good idea!  I want to add that too, but I'm kinda busy right now.  It'll be coming soon!
